google_compute_engine/metadata_scripts/tests/script_retriever_test.py

```diff
@@ -34,6 +34,18 @@ def setUp(self):
         self.mock_logger, self.script_type)
 
   @mock.patch('google_compute_engine.metadata_scripts.script_retriever.subprocess.check_call')
+  def testDownloadGsNoExec(self, mock_call):
+    mock_call.side_effect = subprocess.CalledProcessError('foo', 'bar')
+    gs_url = 'gs://fake/url'
+    self.assertIsNone(self.retriever._DownloadGsUrl(gs_url, self.dest_dir))
+    mock_call.assert_called_once_with(
```

Comment:
OK, process would be much easier if indentations were consistent, 4 or 2, although for old people like me 4 is certainly nicer.